cfgov/jinja2/v1/owning-a-home/loan-estimate/index.html

```diff
@@ -38,48 +38,53 @@
         </nav>
     </div>
     <div class="content_wrapper">
-        <div class="form-explainer
-                    content_main
-                    {% if subnav %}col-9{% endif %}">
-            <div class="page-intro content-l">
-                <div class="content-l_col-2-3">
-                    <h1 class="page-title">Loan Estimate Explainer</h1>
-                    <p class="lead">A Loan Estimate tells you important details about a mortgage loan you have requested. Use this tool to review your Loan Estimate to make sure it reflects what you discussed with the lender. If something looks different from what you expected, ask why. Request multiple Loan Estimates from different lenders so you can compare and choose the loan that's right for you.</p>
-                    <div class="block
-                              block__flush-top
-                              block__padded-top
-                              block__flush-bottom
-                              u-right-on-desktop">
-                        {{ social_media.render( {
-                              "twitter_text": "Use the @CFPB's interactive tool to review your Loan Estimate and get definitions for unfamiliar terms.",
-                              "email_title": "Review your Loan Estimate with the CFPB",
-                              "email_text": "Check the details on your mortgage Loan Estimate and get definitions with this interactive tool from the CFPB.",
-                              "email_signature": "-- From the CFPB",
-                              "linkedin_title": "Learn the lingo of Loan Estimates with the @CFPB.",
-                              "linkedin_text": "The @CFPB has a helpful tool to review your mortgage Loan Estimate and get definitions for unfamiliar terms. Via @CFPB"
-                        } ) }}
+            <div class="content_main">
```

Comment:
Extra indentation added here?

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4782/commits/2c86e63a4ef4b0dbb782442355027186875274a2